desc/magnetic_fields.py

```diff
@@ -1846,6 +1853,328 @@ def from_surface(
         )
 
 
+class OmnigenousField(Optimizable, IOAble):
+    """A magnetic field with perfect omnigenity (but is not necessarily analytic).
+
+    Notes
+    -----
+    Doesn't conform to MagneticField API, as it only knows about :math:`|B|` in
+    computational coordinates, not vector B in lab coordinates.
+
+    Parameters
+    ----------
+    L_B : int
+        Resolution of the radial Chebyshev polynomials for magnetic well parameters.
+    M_B : int
+        Number of monotonic spline knots per surface of the magnetic well parameters.
+    L_x : int
+        Resolution of the radial Chebyshev polynomials for the omnigenity parameters.
```

Comment:
maybe say "omigenity mapping parameters" as `x_lmn` appears in `h` which is the intemeidate function mapping boozer and computatioanl coordinates?